config/default/baseline.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
BMO namespace is surely created somewhere already?

Reply:
This was done before the discussion last week, there is a namespace manifest already in config/namespace folder. I'll update this PR and add these changes there. Thanks for pointing out.